src/modules/colorPicker/ColorPickerUI/Views/ColorEditorView.xaml

```diff
@@ -1,212 +1,300 @@
-            <TextBlock Text="&#xEF3C;"
-                       FontFamily="Segoe MDL2 Assets"
-                       FontSize="44"
-                       HorizontalAlignment="Center"
-                       Foreground="{DynamicResource SecondaryForegroundBrush}" />
-            <TextBlock Margin="0,16,0,0"
-                       TextWrapping="Wrap"
-                       TextAlignment="Center"
-                       HorizontalAlignment="Center"
-                       Foreground="{DynamicResource SecondaryForegroundBrush}"
-                       Text="{x:Static p:Resources.No_colors_yet}"/>
+        <!--  Empty history  -->
+        <StackPanel
+            Grid.Row="1"
+            Grid.ColumnSpan="2"
+            Margin="0,-12,0,0"
+            VerticalAlignment="Center"
+            Orientation="Vertical"
+            Visibility="{Binding ColorsHistory.Count, Converter={StaticResource numberToInvertedVisibilityConverter}}">
+            <ui:SymbolIcon FontSize="44" Symbol="Eyedropper20" />
+            <TextBlock
+                Margin="24,16,24,16"
+                HorizontalAlignment="Center"
+                Foreground="{DynamicResource TextFillColorSecondaryBrush}"
+                Style="{StaticResource CaptionTextBlockStyle}"
+                Text="{x:Static p:Resources.No_colors_yet}"
+                TextAlignment="Center"
+                TextWrapping="Wrap" />
         </StackPanel>
 
-        <Border x:Name="CopiedBorderIndicator"
-                Opacity="0"
-                Grid.ColumnSpan="2"
-                HorizontalAlignment="Stretch"
-                VerticalAlignment="Bottom"
-                Height="0"
-                Background="{DynamicResource SystemControlBackgroundAccentBrush}">
 
-            <TextBlock FontSize="16"
-                       Foreground="White"
-                       HorizontalAlignment="Center"
-                       VerticalAlignment="Center"
-                       AutomationProperties.LiveSetting="Assertive"
-                       AutomationProperties.Name="{x:Static p:Resources.Copied_to_clipboard}">
-                <Run Text="&#xE8C8; " FontFamily="Segoe MDL2 Assets"/>
-                <Run Text="{x:Static p:Resources.Copied_to_clipboard}" FontWeight="SemiBold"/>
+        <Border
+            x:Name="CopiedBorderIndicator"
+            Grid.Row="1"
+            Grid.ColumnSpan="2"
+            Height="0"
+            HorizontalAlignment="Stretch"
+            VerticalAlignment="Bottom"
+            BorderBrush="{DynamicResource CardStrokeColorDefaultBrush}"
+            BorderThickness="0,1,0,0"
+            Opacity="0">
+            <Border.Background>
+                <SolidColorBrush Color="{DynamicResource SystemAccentColorPrimary}" />
+            </Border.Background>
+            <TextBlock
+                HorizontalAlignment="Center"
```

Comment:
Can this be a variable? This assumes the font is available, but might not be the case on Windows 10?
(Or is it included with the framework, maybe?)

Reply:
A great catch, wanted to replace that with a symbolicon but forgot about it. Fixed now!